packages/node-cli/src/commands/ManageAccountCommandGenerator.js

```diff
@@ -0,0 +1,271 @@
+const CommandUtils = require("../utils/CommandUtils");
+const NodeTranslationService = require("../services/NodeTranslationService");
+const ManageAccountOutputFormatter = require("./outputFormatters/ManageAccountOutputFormatter");
+const AuthenticationService = require("../core/authentication/AuthenticationService");
+const { ManageAccountActionResult } = require("./actionresult/ManageAccountActionResult");
+const assert = require("assert");
+
+const inquirer = require("inquirer");
+
+const {
+   COMMAND_MANAGE_ACCOUNT: { ERRORS, QUESTIONS, QUESTIONS_CHOICES, MESSAGES },
+   YES,
+   NO,
+} = require("../services/TranslationKeys");
+
+const ACTION = {
+   EXIT: "exit",
+   RENAME: "rename",
+   REMOVE: "remove",
+   REVOKE: "revoke",
+};
+
+const COMMAND = {
+   OPTIONS: {
+      INFO: "info",
+      LIST: "list",
+      REMOVE: "remove",
+      RENAME: "rename",
+      RENAMETO: "renameto",
+   },
+   FLAGS: {
+      NO_PREVIEW: "no_preview",
+      SKIP_WARNING: "skip_warning",
+      VALIDATE: "validate",
+   },
+};
+
+const ANSWERS_NAMES = {
+   SELECTED_AUTH_ID: "selected_auth_id",
+   ACTION: "action",
+   AUTHID: "authId",
+   RENAMETO: "renameto",
+   REMOVE: "remove",
+};
+
+const {
+   showValidationResults,
+   validateAuthIDNotInList,
+   validateAlphanumericHyphenUnderscore,
+   validateFieldHasNoSpaces,
+   validateFieldIsNotEmpty,
+   validateMaximumLength,
+   validateSameAuthID,
+} = require("../validation/InteractiveAnswersValidator");
+
+const PROPERTIES = {
+   INFO: "info",
+   ACCOUNT_INFO: "accountInfo",
+   URLS: "urls",
+};
+
+const DOMAIN = "domain";
+module.exports = class ManageAccountCommandGenerator extends BaseCommandGenerator {
+   constructor(options) {
+      super(options);
+      this._authenticationService = new AuthenticationService(options.executionPath);
+      this._outputFormatter = new ManageAccountOutputFormatter(options.consoleLogger);
+   }
+
+   async _getCommandQuestions(prompt) {
+      const authIDList = await this._authenticationService.getAuthIds(this._sdkExecutor);
+      let answers = await this._selectAuthID(authIDList, prompt);
+      this._outputFormatter.logAccountInfo(answers[ANSWERS_NAMES.SELECTED_AUTH_ID]);
+      const selectedAuthID = answers[ANSWERS_NAMES.SELECTED_AUTH_ID].authId;
+      answers[ANSWERS_NAMES.ACTION] = await this._selectAction(prompt);
+      if (answers[ANSWERS_NAMES.ACTION] == ACTION.RENAME) {
+         answers[ANSWERS_NAMES.RENAMETO] = await this._introduceNewName(prompt, authIDList, selectedAuthID);
+      } else if (answers[ANSWERS_NAMES.ACTION] == ACTION.REMOVE) {
+         answers[ANSWERS_NAMES.REMOVE] = await this._confirmRemove(prompt);
+      }
+
+      return this._extractAnswers(answers);
+   }
+
+   async _selectAuthID(authIDList, prompt) {
+      var authIDs = Object.entries(authIDList).sort();
+      if (authIDs.length <= 0) {
+         throw Error(NodeTranslationService.getMessage(ERRORS.CREDENTIALS_EMPTY));
```

Comment:
throw `new CLIException(...)` or just the message string

Reply:
Fixed